app/src/main/java/com/osfans/trime/settings/PrefMainActivity.kt

```diff
@@ -145,7 +145,7 @@ class PrefMainActivity: AppCompatActivity(),
                             Log.e(FRAGMENT_TAG, "Deploy Exception: $ex")
                         } finally {
                             progressDialog.dismiss()
-                            exitProcess(0)
+//                              exitProcess(0)
```

Comment:
remove this line?

Reply:
I'm not sure whether it will bring new problems, so I comment it  instead of delete